README.md

```diff
@@ -124,28 +124,32 @@ export function detect(source) {
   return source.match(/some-test/i) !== null;
 }
 
-export function parse({source, generateSourceMap}, done) {
-  // Here you convert the source into refract elements.
+export function parse({minim, generateSourceMap, source}, done) {
```

Comment:
@pksunkara in this case `minim` is an input parameter to the function. The code is equivalent to this ES5 code:

```js
function parse(options, done) {
  var generateSourceMap = options.generateSourceMap;
  var minim = options.minim;
  var source = options.source;
  // ...
}

exports.parse = parse;
```